examples/60_search/example_successive_halving.py

```diff
@@ -118,10 +119,11 @@ def get_smac_object(
 
 automl = autosklearn.classification.AutoSklearnClassifier(
     time_left_for_this_task=40,
-    per_run_time_limit=5,
-    tmp_folder='/tmp/autosklearn_sh_example_tmp',
-    output_folder='/tmp/autosklearn_sh_example_out',
+    per_run_time_limit=10,
+    tmp_folder='/tmp/autosklearn_sh_example_tmp_cv',
```

Comment:
I find those names slightly hard to follow. Could you please use `01`, `02` etc as trailing ways to differentiate.